Doxyfile

```diff
@@ -793,7 +793,7 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                =
+EXCLUDE                = libraries/fc/vendor/editline libraries/fc/vendor/secp256k1-zkp libraries/fc/vendor/websocketpp
```

Comment:
I guess it's better to put this in fc's Doxyfile

Reply:
It will not work like that if we want have the fc included in bitshares-core output.
`doxygen` will not read the `DoxyFile` at sub folders.

We have 3 options:

- remove the `DoxyFile` from FC fully.
- add the excludes to `Doxyfile` in FC in case someone is building the documentation only for FC. We still will want to keep the excludes here as we want FC to be part of bitshares-core output. 
- dont modify `Doxyfile` in FC and produce the external modules output if documentation is being generated for FC only.

Reply:
Oh. I think the 2nd and 3rd are both fine.